templates/apps/career-selector.html

```diff
@@ -3,7 +3,7 @@
     <ol class="career-list">
         {{#if loading}}
         <div class="careers-loading">
-            Loading Careers...
+            {{localize "Loading Careers..."}}
```

Comment:
Does not have an entry in `en.json` and should be something like `SHEET.LoadingCareers`